requirements_dev.txt

```diff
@@ -1,4 +1,4 @@
-hikari==2.0.0.dev104
-maturin==0.12.5
-yt-dlp==2021.12.1
 setuptools-rust==1.1.2
+mkinit
+black
+mypy
```

Comment:
No versions + a tool like mkinit should have its own PR

Reply:
I don't think we need a version, let it be the last one
`mkinit` is a useful thing, so I decided to add it`